React/Base/RCTProfile.h

```diff
@@ -123,6 +135,7 @@ RCT_EXTERN void RCTProfileImmediateEvent(NSString *, NSTimeInterval , NSString *
 
 #define RCTProfileImmediateEvent(...)
 
+
```

Comment:
You need to define a void macro for the methods here, otherwise it might break in release mode (It won't in this case, since it's only used in `RCTWebSocketExecutor` and `RCTProfile` itself, and both will be stripped out in release, but since it's public anyone might decide to use it)